### PR TITLE
Build updates

### PR DESCRIPTION
**Pull request by scottferg**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,30 +1,28 @@
...
+        - go get
+        - go get -t
+        - go build
+        - go test -gocheck.v -coverprofile=coverage.txt -covermode=count
+        - cd fetch && go test -v -coverprofile=coverage.txt -covermode=count
+        - cd ..
+        - cat fetch/coverage.txt >> coverage.txt && rm fetch/coverage.txt
+        - cat coverage.txt
+        - export GIT_HASH=$(git rev-parse HEAD)
+        - export CVR_URL="https://cvr.vokal.io/coverage?commit=$GIT_HASH&owner=$REPO_OWNER&repo=$REPO_NAME&coveragetype=cobertura"
```
Comment (foresmac) [inline .drone.yml]: small thing, but you can delete the line that gets the GIT_HASH and replace it about with `$DRONE_COMMIT`.

Comment (scottferg) [inline .drone.yml]: I know I'll fix that up when I get there

**Comment by scottferg** [comment]
@projectweekend @foresmac build is fixed, feel free to take a look

File: .drone.yml (hunk 1 above)
Comment (scottferg) [inline .drone.yml]: Fixed

**Comment by foresmac** [comment]
LGTM so far.

**Comment by projectweekend** [comment]
very nice

File: fetch/fetch.go [hunk 2]
```diff
@@ -9,7 +9,8 @@ import (
 	"os"
 	"strconv"
 	"strings"
-	"vip/store"
+
+	"github.com/vokal/vip/store"
```
Comment (foresmac) [inline fetch/fetch.go:13]: Why are these not relative imports again? I fear my local dev environment needs changed big time.

**Pull request merged**